eng/pipelines/cadl-ranch-typespec.yml

```diff
@@ -0,0 +1,25 @@
+trigger:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - packages/typespec-go/test/cadlranch
+pr: none
+
+steps:
+  - template: ../steps/set-env.yaml
+    parameters:
+      NodeVersion: "18.x"
```

Comment:
@benbp do we need to set these params since they're already defined in `set-env.yaml`?

Reply:
If you're not trying to override the default then no.